### PR TITLE
Interest page

### PR DESCRIPTION
**Pull request by bassam360**
i just wanna make sure that the design is okay so i can start today on functionalities of the page and it will be all done . don't mind commented things i was trying something :)

### CODE REVIEW

File: src/Pages/InterestPage/InterestPage.js [hunk 1]
```diff
@@ -1,4 +1,135 @@
...
+        </div>
+        <div className=" shadow-sm w-100 ">
+          <div>
+            {interest.length === 0
+              ? null
+              : interest.map((el) => {
+                  return (
+                    <TrackerItem
+                      item={el.title}
+                      date="12/12/2000"
```
Comment (JanaKhanji) [inline src/Pages/InterestPage/InterestPage.js]: here u should pass el.deadline

File: src/Pages/InterestPage/trackerItem.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+function TrackerItem({ item, changeItem, deleteItem, date }) {
+  return (
+    <div className="p-3  shadow-sm w-100 flex items-center justify-between ">
+      <div className="flex w-4/6">
+        <div className="flex justify-between w-11/12">
+          <h1 className="font-medium m-2 text-Grey">{item}</h1>
+          <div
+            className={` border m-2 w-6 rounded-full flex justify-center items-center `}
+          >
+            2
```
Comment (JanaKhanji) [inline src/Pages/InterestPage/trackerItem.js]: this should be also a prop representing the completedItems from the trackers in data.js

**Review by JanaKhanji — CHANGES_REQUESTED** [review]

File: src/Pages/InterestPage/trackerItem.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import uncheck from '../../assets/images/uncheck.png';
+import bin from '../../assets/images/delete.png';
+
+function TrackerItem({ item, changeItem, deleteItem, date }) {
```
Comment (JanaKhanji) [inline src/Pages/InterestPage/trackerItem.js]: why are u getting this props "changeItem, deleteItem" here?

Comment (bassam360) [reply src/Pages/InterestPage/trackerItem.js]: it's not needed right now just in case i need it later if not i will remove them

Comment (JanaKhanji) [reply src/Pages/InterestPage/trackerItem.js]: instead of "item" call the prop "title"

File: src/Pages/InterestPage/trackerItem.js (hunk 2 above)
Comment (bassam360) [reply src/Pages/InterestPage/trackerItem.js]: done

File: src/Pages/InterestPage/InterestPage.js (hunk 1 above)
Comment (bassam360) [reply src/Pages/InterestPage/InterestPage.js]: done

**Comment by osamaakb** [comment]
Good job @bassam360, there are small things in the design needs to be fixed
- why do we need both? keep the search one
image
- This should be clickable, I know you are didn't start working on functionality but just wanted to clear it ahead
image
- This shouldn't be an input but rather just a `p` that when you click you can add new collection
image
- same here
image

**Comment by osamaakb** [comment]
Not sure if this is done or not but there is still functionality needs to be done:
- make the add new tracker works, it should show the form to add a new tracker
- make the add new item work
- the deleted item button works
- the check button works, it should add a line-through the item title when checked
- Search should work too
- when you click on a tracker you should have it's items shown on the left

**Pull request closed without merging**